source/client/versus.html

```diff
@@ -5,7 +5,8 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
   <title>Tarot, but a Game | Versus</title>
-
+  
+  <meta id="current_world_event" name="current_world_event" content="none">
```

Comment:
Why is this a meta tag?

Reply:
for e2e testing purposes until the frontend is developed

Reply:
Fair enough